data/us/california/san-diego.json

```diff
@@ -17,21 +17,20 @@
     "tags": []
   },
   {
-    "id": "brewery-in-planning-san-diego",
-    "name": "Brewery in Planning - San Diego",
-    "brewery_type": "planning",
-    "street": "",
+    "name": "Gravity Heights",
+    "brewery_type": "micro",
+    "street": "9920 Pacific Heights Blvd",
     "city": "San Diego",
     "state": "California",
-    "postal_code": "92121-4396",
+    "postal_code": "92121",
     "website_url": "http://WWW.gravityheights.com",
-    "phone": "8588882188",
+    "phone": "+1 858 283 8206",
     "created_at": "2018-07-24 01:32:53.189216",
-    "updated_at": "2018-08-23 23:26:06.455378",
+    "updated_at": "2020-01-31",
```

Comment:
Thinking about it more, this field shouldn't exist as it should get updated automatically.